mitodl/mitxonline
# MicroMasters data migration command and model

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #721

#### What's this PR do?

- Adds data models for mapping course and program ids
- Adds a management command that runs SQL files in alphanumeric order by filename.

#### How should this be manually tested?

- You should be able to go to django-admin and add new records for `CourseId` and `ProgramId`
- You should be able to run the `./manage.py import_micromasters_data` command
